hkmc2Benchmarks/src/test/bench/FingerTreesAsStacks.mls

```diff
@@ -0,0 +1,195 @@
+:js 
```

Comment:
Everything in this file is silent... Just put `:silent` at the top, here, where it becomes global.
```suggestion
:js
:silent
```